enterprise/app/models/enterprise/audit/account.rb

```diff
@@ -2,6 +2,7 @@ module Enterprise::Audit::Account
   extend ActiveSupport::Concern
 
   included do
+    audited
```

Comment:
let's limit this to update alone for now.

Reply:
done